umm-spec-lib/src/cmr/umm_spec/migration/temporal_range_type_migration.clj

```diff
@@ -0,0 +1,9 @@
+(ns cmr.umm-spec.migration.temporal-range-type-migration
+  "Contains helper functions for migrating between different versions of UMM temporal range type."
+  (:require
+   [cmr.common.util :as util :refer [update-in-each]]))
+
+(defn migrate-up
+  "Migrate temporal range type from existing to non-existing."
+  [c]
+  (update-in-each c [:TemporalExtents] dissoc :TemporalRangeType)) 
```

Comment:
I don't think we need a separate file for this, it can go in the migration function.